### DISCUSSION TITLE
labgrid-suggest: no suggested match found (no ID_PATH)

### DISCUSSION THREAD

**Discussion by Mdfayaz003** [labgrid-project/labgrid#1104] [Q&A] [answered]
I tried labgrid on different **Ubuntu versions (18.04, 20.04, 22.04).**
On execution of `labgrid-suggest`, it lists all **USB devices** with its `ID_PATH` on both the `20.04` and `22.04` versions of Ubuntu. But on **Ubuntu 18.04**, it lists all USB devices but not its `ID_PATH`, instead, it shows as `no suggested match found`

Is there any Ubuntu version problem or any packages to be installed for the abovementioned problem?

Can you share any documentation regarding `labgrid-suggest` working for a better understanding?

**Comment by Bastian-Krause** [comment, marked as answer]
Did you create the udev rule mentioned in https://labgrid.readthedocs.io/en/latest/configuration.html#udev-matching ?

**Reply by Mdfayaz003** [reply]
Thanks, @Bastian-Krause for your reply.

But I neither **created the udev rule** on `Ubuntu 20.04` nor on `Ubuntu 22.04`, but it showed the `ID_PATH`.
Unable to see it on `Ubuntu 18.04`

**Reply by Bastian-Krause** [reply]
I have no overview on which distro the rule is required. Please try adding the rule. If the `ID_PATH` still does not show up, comment here again.

**Reply by Mdfayaz003** [reply]
Thanks, @Bastian-Krause
I try to add rules and update you soon

**Reply by Mdfayaz003** [reply]
Thanks @Bastian-Krause
It worked after updating `udev` rule. Thanks a lot for you support.
